README.md

```diff
@@ -143,69 +264,114 @@ Implementing push notifications requires a few additional snippets of code to en
 	    }
 
 	    // Enable or disable features based on the authorization status.
+	    // if you didn't register for remote notifications above you can call `registerForRemoteNotifications` here
+	    // Klaviyo SDK will automatically update the authorization status on next app launch
 	}
 
-	UIApplication.shared.registerForRemoteNotifications()
+    return true
+}
 ```
 
-2. Add the following code to the application delegate file in `application:didRegisterForRemoteNotificationsWithDeviceToken`. You may need to add this code to your application delegate if you have not done so already.
+### Receiving Push Notifications
 
+#### Tracking Open Events
+
+Implement the [`userNotificationCenter:didReceive:withCompletionHandler`](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter)
+and [`userNotificationCenter:willPresent:withCompletionHandler`](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649518-usernotificationcenter) to handle push notifications received when the app is in the background and foreground.
+method in your app delegate to track when a user opens a push notification. This will handle tracking opens for when the app is backgrounded and the user taps on the notification.
```

Comment:
Seems the start of this sentence is missing?

Reply:
Not really, I was just trying to get to the point directly but I did notice this read weird so changed it to include some context like - 
When a user taps on a push notification, Implement  [`userNotificationCenter:didReceive:withCompletionHandler`](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter)
and [`userNotificationCenter:willPresent:withCompletionHandler`](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649518-usernotificationcenter) in your application delegate to handle receiving push notifications 
when the app is in the background and foreground respectively.